packages/wrangler/src/dev/dev.tsx

```diff
@@ -103,6 +104,13 @@ export function DevImplementation(props: DevProps): JSX.Element {
     nodeCompat: props.nodeCompat,
   });
 
+  // During console resize UI was tearing and leaving artifacts,
+  // this is a workaround that clears the tearing artifacts
+  stdout?.on("resize", () => {
```

Comment:
Does this mean this would clear out anything logged in the terminal? That doesn't seem like a good fix.

Reply:
specifically, anything that the actual worker would have logged, requests, etc, those would all get cleared, and I don't think that's good.

Reply:
Yeah, just the first solution I figured out. I thought the clearing current console context wouldn't be best either (mentioned it in the commit) but got the ball rolling anyway.